app/views/income_before_tax.scala.html

```diff
@@ -39,68 +39,79 @@
 @includes.taxs_main(title = Messages("ats.income_before_tax.html.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo), viewModel.utr, contentHeader = Option(headerContent), actingAttorney = actingAsAttorney) {
 
     @helpers.back_link(Messages("generic.back"))
-    <h1>
-        <span class="gray heading-medium block-heading">
-          @Html(Messages("generic.tax_year_heading", viewModel.taxYearFrom, viewModel.taxYearTo))
-        </span>
-      @Html(Messages("ats.income_before_tax.title"))
-    </h1>
-    <p class="lede">@Html(Messages("ats.income_before_tax.intro"))                    </p>
+
+    @if(!viewModel.isPaye){
```

Comment:
`if/else` in view which could likely be avoided with refactor

Reply:
Yes. Some options here... Please specify how you think we should do this. Thx.